frontend/src/index.js

```diff
@@ -0,0 +1,107 @@
+import * as React from "react";
+import { createRoot } from "react-dom/client";
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  Link,
+  Outlet,
+} from "react-router-dom";
+import Navbar from "./components/Navbar";
+
+import Home from "./routes/Home";
+import About from "./routes/About";
+import Signup from "./routes/Signup"
+
+const AppLayout = () => {
+	return (
+		<>
+		<Navbar />
+		<Outlet />
```

Comment:
also, foloseste typescript, nu javascript 😄 , aevai deja autogenerat in index.tsx - ala e mounting point-ul pentru componentele de react